cmd/wallet-sdk-gomobile/docs/usage.md

```diff
@@ -234,7 +230,7 @@ import dev.trustbloc.wallet.sdk.localkms.MemKMSStore
 val memKMSStore = MemKMSStore.MemKMSStore()
 val kms = Localkms.newKMS(memKMSStore)
 val didCreator = Creator(kms as KeyWriter)
-val didDocResolution = didCreator.create("key", CreateDIDOpts()) // Create a did:key doc
+val didDocResolution = didCreator.create("key", null) // Create a did:key doc
```

Comment:
I understand we internally create default opts in case of null, but can we have examples with Opts passed to the API? Same comments for other examples.

Reply:
Sure - but I think I should do it in a follow-up PR, since it's a little bit out of scope and this PR has already gotten pretty huge.